content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -52,7 +52,7 @@ You can configure a static Pod with either a [file system hosted configuration f
 
 ### Filesystem-hosted static Pod manifest {#configuration-files}
 
-Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [Kubelet Configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
+Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
```

Comment:
@helight , thanks for the updates.
What do you think about these possible edits for lines 55 - 56:

Manifests are standard Pod definitions in JSON or YAML format.
You can define a static Pod by creating a [kubectl configuration file](/docs/tasks/administer-cluster/kubelet-config-file) and setting the `staticPodPath` field to a directory containing your static Pod manifests.
The kubelet periodically scans this directory and creates or deletes the static Pods as the directory contents change.

Reply:
so, what for? I don't thinks it is fix any problem. it has the same meaning with origin words.

Reply:
hello @helight. As a native English speaker/reader, I found the second sentence somewhat unclear.
I tried to answer the question: What (which) periodically scans the directory and creates ...

Reply:
yes, you are right about the sentence. it should make clear that kubelet scan the file.
but in fact I can understand what this origin sentence talk about, I know the [kubelet configuration file] should be scaned by kubelet.
so I not sure it should be modified or not.

Reply:
@kbhawkey I think your proposed wording will be easier for localization teams to work with too.

Reply:
@sftim so are you agree we should modified it as @kbhawkey 's fix

Reply:
@helight, your changes seem fine. If time, I will create another PR to modify the sentence structure. Thanks!

Reply:
@kbhawkey ok, thanks.
 If there is anything I can do, just tell me, I am glad to do that.